CONTRIBUTING.md

```diff
@@ -29,6 +29,16 @@ We follow the [Google Java Style
 Guide](https://google.github.io/styleguide/javaguide.html). Our
 build automatically will provide warnings for simple style issues.
 
+Run the following command to format all files:
```

Comment:
Suggestion: You could also mention that this runs google-java-format and give a link.

Reply:
SG, done.